docs/networking/split-horizon_config.md

```diff
@@ -0,0 +1,117 @@
+    - **Namespace**: The Kubernetes namespace where you want to create the policy.
+    - **Domain**: A domain represents how your database endpoint is identified (for example, mydb.everest.local).
+    - **Certificate**: Upload your **Transport Layer Security 
+    (TLS)** certificate. 
+    For information on how to obtain a TLS certificate, refer to the [TLS Certificate page](split-horizon_create_CA_certs.md).
+    - **key**: Upload your Private key file. The private key pairs with the TLS certificate and is used to establish encrypted connections.
+    - **Ca cert**: Upload your Certificate Authority (CA) Certificate. A CA certificate is the root or intermediate certificate from the Certificate Authority (CA) that signed your TLS certificate. It helps clients verify that your database’s certificate is valid and trustworthy.
+
+5. Click **Create**. Your Split-Horizon DNS policy will be created and appears in the configuration list.
+
+    ![!image](../images/split-horizon-policies.png)
+
+
+## Edit split-horizon DNS policy
+
+Here's how you can edit a Split-Horizon DNS policy for your MongoDB cluster:
+{.power-number}
+
+1. Navigate to the Percona Everest home page and go to <i class="uil uil-cog"></i> **Settings > Policies & configurations**.
+
+2. In the **Split-Horizon DNS** section, click **Configure**. The **Split-Horizon DNS** page opens. Here, you can see the existing Split-Horizon DNS policies.
+
+3. Click the ellipsis **(⋮)** next to the Split-Horizon DNS policy you want to modify and select **Edit**. The **Edit configuration** pop-up opens.
+
+    ![!image](../images/split-horizon-edit-policies.png)
+
+    !!! note
+        You can edit a Split-Horizon DNS policy only if it is not associated with any active cluster.
+
+4. Update the desired fields.
+
+    !!! note
+        You can only edit the **Domain, Ca cert, and Ca key** fields. 
+
+5. Click **Save** to apply the changes.
+
+
+## Delete a Split-Horizon DNS policy
+
+ Here's how you can delete an existing Split-Horizon DNS policy for a MongoDB cluster:
+{.power-number}
+
+1. Navigate to the Percona Everest home page and go to <i class="uil uil-cog"></i> **Settings > Policies & configurations**.
+
+2. In the **Split-Horizon DNS** section, click **Configure**. The **Split-Horizon DNS** page opens. Here, you can see the existing Split-Horizon DNS policies.
+
+3. Click on the ellipsis **(⋮)** next to the Split-Horizon DNS policy you want to delete. 
+
+    ![!image](../images/split-horizon-delete%20configuration.png)
+
+
+4. Click **Delete**. A confirmation pop-up will be displayed for deleting the Split-Horizon DNS config policy.
+
+    ![!image](../images/split-horizon-delete_configuration_confirmation.png)
+
+5. Enter the **Config name** in the text box to confirm the deletion, and click **Delete**. Your Split-Horizon DNS policy will be deleted.
+
+    ![!image](../images/split-horizon-delete%20config_box.png)
+
+
+## Configure Split-Horizon DNS policy for your MongoDB cluster
```

Comment:
important note here: it is applicable to non-shared clusters only for now

Reply:
Added